x11-libs/libxcursor/libxcursor-1.1.15.recipe

```diff
@@ -35,30 +38,28 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:cursorproto$secondaryArchSuffix
```

Comment:
I can't find any link to cursorproto for this one?

Reply:
I though I had created a recipe which provided a `devel:cursorproto$secondaryArchSuffix` but it turns out that there isn't any tarball that I could have used in such a recipe, so I guess this line was an accident.
Anyway, without this line the recipe builds fine, so I fixed it in d977b5c2717dbdd33f972040429555bb56d310ff.

Reply:
Thanks :) although now I'm of par with this PR ;)